docs/graphs/graph-queries/graph-keywords/prune.md

```diff
@@ -0,0 +1,58 @@
+---
+title: PRUNE
+---
+
+The PRUNE keyword in [C8QL](../../../queries/c8ql/) is used in conjunction with graph traversal queries to optimize performance by controlling the traversal process. It helps to reduce the number of paths explored during the traversal by stopping further exploration of a path if a specified condition is met.
+
+By using `PRUNE`, you can define criteria that, once met, will prevent the traversal from continuing along a particular path. This early termination of traversal can significantly improve query performance, especially in large graphs, by eliminating unnecessary exploration of nodes and edges that do not contribute to the desired result.
+
+For more information about using `PRUNE`, refer to [Pruning for Efficient Searches](../traversal-queries/traversal-pruning).
+
+## Purpose
+
+The purpose of the `PRUNE` keyword is to control the traversal process by filtering out paths or branches during the graph traversal that do not meet specified conditions. This allows users to efficiently explore specific parts of a graph by pruning unwanted paths early in the traversal, reducing the overall complexity and improving query performance.
```

Comment:
Maybe also add that helps manage resources such as memory and CPU usage more effectively.

Reply:
Updated. What do you think?